CHANGELOG.md

```diff
@@ -188,6 +189,9 @@ binary | sha256 hash
 
 **No notable changes for this release**
 
+## Known issues
+
+* master newer than v1.4.0 has backward compatible issues with older kubectl. Specifically, if "kubectl rolling-update" does not change the name of the replication controller (RC), the command will fail to create the new RC and report the RC already exists. When this happens, the pods are successfully updated, but the name of the RC is its original name plus a hash value. If you want to rename the RC, please download a kubectl newer than v1.4.0, then run rolling-update to change the name.
```

Comment:
I'm going to help out with the wording here-- I've started something better below. Also, please start a section specifically about the behavior changes the garbage collector introduces and put this there.

```
`kubectl` 1.3's rolling-update command is compatible with Kubernetes 1.4 and higher **only if** you specify a new replication controller name. You will need to update to kubectl 1.4 or higher to use the rolling update command against a 1.4 cluster if you want to keep the original name, or you'll have to do two rolling updates.

If you do happen to use kubectl 1.3's rolling update against a 1.4 cluster, it will fail, usually with an error message that will direct you here. If you saw that error, then don't worry, the operation succeeded except for the part where the new replication controller is renamed back to the old name. You can just do another rolling update to change the name back: look for a replication controller that has the original name plus a random suffix.

Unfortunately, there is a second possible failure mode: <describe failure mode>

If this happens to you, then <describe fix>

```